### ISSUE TITLE
stats plugin not returning all actions

### ISSUE THREAD

**Issue by esatterwhite** [hemerajs/hemera#217]
## Description
I 'm using the stats plugin in various services and trying to get all of the registered actions, but it seems to only return the actions from a one of the services in a cluster. I'll only get the actions from Service A, or Service B, or Service C rather than the actions for Service A + B+ C.

## Expected Result

actions from every service are returned

## Actual Result

actions list are from one and only one of the connected services

## Context

- [x] nats-hemera (core)
- [ ] hemera-zipkin
- [ ] hemera-store
- [x] hemera-stats
- [ ] hemera-rabbitmq
- [ ] hemera-nsq
- [ ] hemera-elasticsearch
- [x] hemera-joi
- [ ] hemera-msgpack
- [ ] hemera-avro
- [ ] hemera-jwt-auth

## Your Environment
- NATS: 1.0.0
- Hemera: 5.8.x
- NodeJs: 8+

**Comment by StarpTech** [comment]
Hi @esatterwhite if you call
```
hemera.act(
  {
    topic: 'stats',
    cmd: 'registeredActions'
  },
  function(err, resp) {
    // is called multiple times
  }
)
```
you will receive multiple responses with the different actions from the services.

**Comment by esatterwhite** [comment]
Oh interesting. Is there a way to know when all of the services have responded?

**Comment by StarpTech** [comment]
This a pull mechanism. If you think that all service should send their information to a specific subscriber at startup, feel free to create PR.

**Comment by StarpTech** [comment]
> Oh interesting. Is there a way to know when all of the services have responded?

You can't know it. You are able to start new services at any time. I prefer to implement a solution as suggested above.

**Comment by StarpTech** [comment]
@esatterwhite I implemented here https://github.com/hemerajs/hemera/pull/218 please review

**Comment by StarpTech** [comment]
After the PR you can just call

```js
hemera.ready(() => {
	hemera.sendActionStats({
	  topic: 'process.stats'
	})
})
```
to inform your subscriber about new updates. This requires of course that your service `process.stats` is ready before the others.

**Comment by StarpTech** [comment]
Feature was published with hemera-stats@5.2.0
